agent.go

```diff
@@ -1161,6 +1161,11 @@ func newContainerCb(pod *pod, data []byte) error {
 		return err
 	}
 
+	dest := filepath.Join(absoluteRootFs, "/dev")
+	if err := syscall.Mount("tmpfs", dest, "tmpfs", syscall.MS_NOSUID|syscall.MS_STRICTATIME, ""); err != nil {
```

Comment:
Same here, I think we should umount that one when deleting the container.

Reply:
@amshinde @sboeuf should we not setup a /dev directory per container. Not all containers may be allowed to see all mount points.

Reply:
As discussed offline, this should be gated by the fact that the user has requested a mount `/dev/:/dev`. This is the way we would do that with runc, and this is the proper way to make sure the user expects to see new devices being created and to be able to get access to them.

Reply:
This raises the question of handling special cases like this one, where cannot expect to mount the directory from the host. Should we maintain a list of such directories ? Actually, we should define such a list first and see how we plan to handle every case.

Reply:
@sboeuf See this commit where I have addressed `-v /dev:/dev` : https://github.com/clearcontainers/agent/pull/180/commits/0680f5a626c85ad5daf299e2bc1903b945ce1a5d